From abc2125fad51b2a3889248412ecc654c38d2efbb Mon Sep 17 00:00:00 2001
From: "d.marlin" <dmarlin@redhat.com>
Date: Wed, 27 Aug 2014 16:26:45 -0400
Subject: [PATCH 05/15] Allow aarch64 in the rhel build macros.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/macros.pesign | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/macros.pesign b/src/macros.pesign
index 7faa4a1..39374ce 100644
--- a/src/macros.pesign
+++ b/src/macros.pesign
@@ -21,7 +21,9 @@
 # -a <input ca cert filename>		# rhel only
 # -s 					# perform signing
 %pesign(i:o:C:e:c:n:a:s)						\
-  if [ -x %{_pesign} -a "%{_target_cpu}" == "x86_64" ]; then		\
+  if [ -x %{_pesign} ] &&  						\\\
+       [ "%{_target_cpu}" == "x86_64" -o 				\\\
+         "%{_target_cpu}" == "aarch64" ]; then				\
     if [ "0%{?rhel}" -ge "7" -a -f /usr/bin/rpm-sign ]; then		\
       nss=$(mktemp -p $PWD -d)						\
       echo > ${nss}/pwfile						\
-- 
2.5.5

